test/newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Staybnb Reviews'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '6c1c636d00a7d0befcf706544895b3df03071a01',
```

Comment:
this file shouldn't be on git, you'll likely get an email that you uploaded this key.  not the biggest deal since it's for a free trial, but this file should be ignored

Reply:
included in .gitignore